recipes/lz4/all/test_package/conanfile.py

```diff
@@ -8,6 +8,7 @@ class TestPackageConan(ConanFile):
 
     def build(self):
         cmake = CMake(self)
+        cmake.definitions["TEST_SHARED_LIB"] = dict(self.options["lz4"].items()).get("fPIC", True)
```

Comment:
Didn't this work? Shouldn't this be `.shared` instead of `.fPIC`? 🤔 
```suggestion
        cmake.definitions["TEST_SHARED_LIB"] = self.options["lz4"].fPIC
```

Reply:
`fPIC` is removed on Windows targets. So it would return always `None`.

Also, this recipe does not remove `fPIC` when `shared=True`.
Should fix that.

Reply:
Ok, now I understand the trick. It should use shared anyway

Reply:
This line in `test_package` is fine, I think.
It is `True` by default and only False when the `fPIC` option is `False`.